### PR TITLE
feat(flex): Support using property of string type as primary key for vertex

### PR DESCRIPTION
**Pull request by liulx20**
## What do these changes do?

oid support string_view

## Related issue number

Fixes

### CODE REVIEW

File: flex/engines/graph_db/database/single_vertex_insert_transaction.cc [hunk 1]
```diff
@@ -162,17 +162,22 @@ void SingleVertexInsertTransaction::ingestWal() {
...
           graph_.add_vertex(added_vertex_label_, added_vertex_id_);
       graph_.get_vertex_table(added_vertex_label_)
           .ingest(added_vertex_vid_, arc);
     } else if (op_type == 1) {
+      Any temp;
       label_t src_label, dst_label, edge_label;
       arc >> src_label;
-      arc.GetBytes(sizeof(oid_t));
+      arc >> temp;
+      // arc.GetBytes(sizeof(PropertyType) + sizeof(int64_t));
```
Comment (zhanglei1949) [inline flex/engines/graph_db/database/single_vertex_insert_transaction.cc:177]: These comments can be removed?

File: flex/engines/graph_db/app/server_app.cc [hunk 2]
```diff
@@ -22,7 +22,7 @@ static uint32_t get_vertex_vid(const gs::ReadTransaction& txn, uint8_t label,
   uint32_t vid = std::numeric_limits<uint32_t>::max();
   auto vit = txn.GetVertexIterator(label);
   for (; vit.IsValid(); vit.Next()) {
-    if (vit.GetId() == id) {
+    if (vit.GetId().AsInt64() == id) {
```
Comment (zhanglei1949) [inline flex/engines/graph_db/app/server_app.cc:25]: If the result of `vit.GetId()` must be `int64()`, do the check first?

File: flex/engines/graph_db/app/server_app.cc [hunk 3]
```diff
@@ -22,7 +22,7 @@ static uint32_t get_vertex_vid(const gs::ReadTransaction& txn, uint8_t label,
   uint32_t vid = std::numeric_limits<uint32_t>::max();
```
Comment (zhanglei1949) [inline flex/engines/graph_db/app/server_app.cc:22]: `using oid_t = int64_t;` declared in `flex/storages/rt_mutable_graph/types.h` should be removed, and we shall ensure can remove all usage of oid_t is this PR?

File: flex/engines/graph_db/app/server_app.cc (hunk 2 above)
Comment (liulx20) [reply flex/engines/graph_db/app/server_app.cc:25]: Checked inside the AsXX() function

**Review by luoxiaojian — APPROVED** [review]

**Review by luoxiaojian — APPROVED** [review]

**Review by zhanglei1949 — APPROVED** [review]
LGTM

**Pull request merged**